packages/calcite-components/src/tests/utils.ts

```diff
@@ -268,7 +268,7 @@ export async function newProgrammaticE2EPage(): Promise<E2EPage> {
  */
 export async function skipAnimations(page: E2EPage): Promise<void> {
   await page.addStyleTag({
-    content: `:root { --calcite-duration-factor: 0; }`
+    content: `:root { --calcite-duration-factor: 0.01; }`
```

Comment:
Can you explain why changing this to 0.1 is required, is it a chromatic thing?

Reply:
Sure. It's to work around [this issue](https://github.com/Esri/calcite-components/issues/6604).

Good catch though. I'll add a comment to this for context.